css/v202209/symbiota/main.css

```diff
@@ -324,6 +324,47 @@ table.styledtable tr.alt td {
   }
 }
 
+.nested-li {
+  margin-left: 1.5em;
+}
+
+.fieldset-like {
+  /* borrowed from: https://www.geeksforgeeks.org/how-to-achieve-fieldset-like-effect-without-using-fieldset-tag/ */
+  border: 2px solid;
+  border-top: none;
+  margin: 32px 2px;
```

Comment:
Have you looked at using rem instead of px? Not sure what is recommended for accessibility but this article has some insights on I though were helpful [em-vs-rem-vs-px](https://engageinteractive.co.uk/blog/em-vs-rem-vs-px)

Reply:
Good call. Thanks! Fixed in eb9c768d4